vendor/VRControls.js

```diff
@@ -35,9 +35,9 @@ THREE.VRControls = function ( object, onError ) {
 
 	}
 
-	if ( navigator.getVRDisplays ) {
+	if ( window.navigator.getVRDisplays ) {
```

Comment:
Because VRControls is a third-party three.js module, I'd go to utils.device and change it so that those modules aren't instantiated until they're needed.

```js
var controls;

someUtilFunction () {
  if (!controls) { instantiateControls(); }  // Lazy-instantiate VRControls to not break in Node.
}
```